src/Sulu/Bundle/SnippetBundle/Resources/js/views/SnippetAreas/tests/SnippetAreas.test.js

```diff
@@ -27,10 +34,7 @@ test('Show loader when loading snippet areas', () => {
     const SnippetAreas = require('../SnippetAreas').default;
     const SnippetAreaStore = require('../stores/SnippetAreaStore');
 
-    const router = new Router({});
-    router.attributes = {
-        webspace: 'sulu',
-    };
```

Comment:
Why are the `attributes` removed in all these tests? Haven't they been necessary before as well?